app/src/analysis/initializeCoreDb.ts

```diff
@@ -72,24 +76,29 @@ export async function initializeCoreDb(): Promise<sqlite3.Database> {
   }
   const possibleAddressBookDB = await findPossibleAddressBookDB();
   const lorDB = initializeDB(chatPaths.app);
+
+  // Drop everything LOR specific if it exists
   await dropAllTables(lorDB);
 
-  log.info('Dropped all tables.');
+  // Add the contact name column regardless
+  // It will just be empty if we don't find an address book db table
+  // As a result, we can use COALESCE(contact_name, id)
+  await addContactNameColumn(lorDB);
 
-  // Create contact table
+  // If we found an address book table, let's create a contact_table and attach it to the main lorDB.
   if (possibleAddressBookDB) {
     try {
+      // Create contact table takes the possibleAddressBookDB not the LOR DB
       await new ContactTable(
-        lorDB,
+        possibleAddressBookDB,
```

Comment:
@Teddarific FYI for future reference, `new Contact()` takes the address book db, not the chat.db, hence why you were getting that error of ZADRESSBOOK not found